source/configure/site-configuration-settings.rst

```diff
@@ -1091,6 +1100,94 @@ Message priority
+
++---------------------------------------------------------------+-------------------------------------------------------------------------------------------+
+| The number of minutes between repeated notifications for      | - System Config path: **Site Configuration > Posts**                                      |
+| urgent messages sent with persistent notifications.           | - ``config.json`` setting: ``.ServiceSettings.PersistentNotificationIntervalMinutes: 5``  |
+|                                                               | - Environment variable: ``MM_SERVICESETTINGS_PERSISTENTNOTIFICATIONINTERVALMINUTES``      |
+| Numerical input. Default is **5**.                            |                                                                                           |
++---------------------------------------------------------------+-------------------------------------------------------------------------------------------+
+
+.. config:setting:: posts-totalnumberofpersistentnotificationsperpost
+  :displayname: Total number of persistent notifications per post (Posts)
+  :systemconsole: Site Configuration > Posts
+  :configjson: .ServiceSettings.PersistentNotificationMaxCount
+  :environment: MM_SERVICESETTINGS_PERSISTENTNOTIFICATIONMAXCOUNT
+  :description: The maximum number of times users may receive persistent notifications. Default is **6**.
+
+Total number of persistent notifications per post
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++-------------------------------------------------------------+------------------------------------------------------------------------------------+
+| The maximum number of times users may receive persistent    | - System Config path: **Site Configuration > Posts**                               |
+| notifications.                                              | - ``config.json`` setting: ``.ServiceSettings.PersistentNotificationMaxCount: 6``  |
+|                                                             | - Environment variable: ``MM_SERVICESETTINGS_PERSISTENTNOTIFICATIONMAXCOUNT``      |
+| Numerical input. Default is **6**.                          |                                                                                    |
++-------------------------------------------------------------+------------------------------------------------------------------------------------+
+
+.. config:setting:: posts-persistentnotificationsforguestusers
+  :displayname: Persistent notifications for guest users (Posts)
+  :systemconsole: Site Configuration > Posts
+  :configjson: .ServiceSettings.AllowPersistentNotificationsForGuests
+  :environment: MM_SERVICESETTINGS_ALLOWPERSISTENTNOTIFICATIONSFORGUESTS
+
+  - **true**:  Guest users can trigger repeating notifications to  mentioned recipients of urgent messages.
+  - **false**: **(Default)** Disables the ability for guest users to send repeating notifications.
+
+Persistent notifications for guest users
```

Comment:
@esethna - This particular config setting isn't visible in the System Console intentionally, correct?

Reply:
Hmm, this setting was never built afaik